testing/cli/test_cli_sct_compute_compression.py

```diff
@@ -0,0 +1,155 @@
+    with pytest.raises(SystemExit) as e:
+        sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-l', dummy_3d_label_label, '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_compute_compression_check_missing_input_l(tmp_path, dummy_metrics_csv, dummy_metrics_csv_pam50):
+    """ Run sct_compute_mscc when missing -l"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-i-PAM50', dummy_metrics_csv_pam50, '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_compute_compression_check_wrong_sex(tmp_path, dummy_3d_label_label, dummy_metrics_csv, dummy_metrics_csv_pam50):
+    """ Run sct_compute_compression when missing -l"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-i-PAM50', dummy_metrics_csv_pam50, '-l', dummy_3d_label_label,
+                                           '-sex', 'J', '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_compute_compression_check_wrong_age(tmp_path, dummy_3d_label_label, dummy_metrics_csv, dummy_metrics_csv_pam50):
+    """ Run sct_compute_compression when missing -l"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-i-PAM50', dummy_metrics_csv_pam50, '-l', dummy_3d_label_label,
+                                           '-age', '20', '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_compute_compression_check_wrong_metric(tmp_path, dummy_3d_label_label, dummy_metrics_csv, dummy_metrics_csv_pam50):
+    """ Run sct_compute_compression when missing -l"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-i-PAM50', dummy_metrics_csv_pam50, '-l', dummy_3d_label_label,
+                                           '-metric', 'MEAN', '20', '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_compute_compression(tmp_path, dummy_3d_label_label, dummy_metrics_csv, dummy_metrics_csv_pam50):
+    """ Run sct_compute_compression and chexk mscc and normalized mscc"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    sct_compute_compression.main(argv=['-i', dummy_metrics_csv, '-i-PAM50', dummy_metrics_csv_pam50, '-l', dummy_3d_label_label,
+                                       '-o', filename])
+    with open(filename, "r") as csvfile:
+        reader = csv.DictReader(csvfile, delimiter=',')
+        row = next(reader)
+        assert row['Compression Level'] == '7.0'
+        assert float(row['MEAN(diameter_AP) ratio']) == pytest.approx(11.471813876181447)
+        assert float(row['Normalized MEAN(diameter_AP) ratio']) == pytest.approx(11.372089241980499)
```

Comment:
I am a bit reluctant of testing the output values of the normalization since it depends on the control data.. Each time we will update https://github.com/spinalcordtoolbox/data_normalize_metric_pam50/ the test will need to be modified. I think I would remove it. @joshuacwnewton what do you think?

Reply:
Sorry for the very late reply to this older comment! I missed it until just now.

I think it's okay to test these values? 

The reason I think so is because,  every time we update the `data_normalize_metric_pam50` repo, we would need to create a new release anyway for the dataset, and then update the link in `sct_download_data`?

So, we would already be creating a PR in SCT, and I don't think it's too much extra work to update the values at the same time.